<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdaterVersionConfigGenerator</class>
 <widget class="QDialog" name="UpdaterVersionConfigGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpdaterVersionConfigGenerator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>UpdaterPath:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="SelectBtn">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Generate">
     <property name="text">
      <string>Generate Updater Version Config</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
